apm/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,45 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+const routes: Routes = [
+    {
+        path: 'welcome',
+        loadChildren: () => import('./welcome/welcome.module').then(m => m.WelcomeModule)
+    },
+    {
+        path: 'products',
+        loadChildren: () => import('./products/product.module').then(m => m.ProductModule)
+    },
+    {
+        path: 'products/:id',
+        loadChildren: () => import('./products/product.module').then(m => m.ProductModule)
```

Comment:
ProductModule is lazy loaded twice. This should not be the case, instead you should be having the route within the ProductModule

Reply:
Looking at the Products module i think this could be removed